File: datasets/IR-Plag/original-kotlin-rest-java/case-06/plagiarized/L5/05/singlearray.java
package IR

import java.util.*;

class singlearray
{
	static int[] n = new int[10];
	static void print()
	{
				for (int a = 9; a >= 0; a--) 
		{
			System.out.println(n[a]);
		}
	}
	public static void main (String[] args)
	{
				
		Scanner inp = new Scanner(System.in);

		int i=0;
		while (i < 10) 
		{
			System.out.print("Read a number: ");
			n[i] = inp.nextInt();
			i++;
		}
print();
	}
}

File: datasets/IR-Plag/original-java-rest-kotlin/case-06/plagiarized/L2/04/singlearray.kt
package IR

import java.util.*

internal object singlearray {
    @JvmStatic
    fun main(args: Array<String>) {
        val inp = Scanner(System.`in`)
        val n = IntArray(10)
        for (i in 0..9) {
            print("Read a number: ")
            n[i] = inp.nextInt()
        }
        for (a in 9 downTo 0) {
            println(n[a])
        }
    }
}